src/Microsoft.DotNet.Darc/Darc/Operations/VirtualMonoRepo/ResolveConflictOperation.cs

```diff
@@ -0,0 +1,215 @@
+        CancellationToken cancellationToken)
+    {
+        if (string.IsNullOrEmpty(_options.SourceRepo))
+        {
+            throw new ArgumentException("Please specify a local path on disk to the source git repo" +
+                " or VMR to flow from.");
+        }
+
+        var subscription = await FetchCodeflowSubscriptionAsync(_options.SubscriptionId);
+
+        var pr = await _pcsApiClient.PullRequest.GetTrackedPullRequestBySubscriptionIdAsync(
+            subscription.Id.ToString(),
+            cancellationToken);
+
+        ValidateConflictingPrAsync(pr);
+
+        var buildId = GetBuildIdFromTrackedPr(pr, subscription.Id);
+
+        var sourceGitRepoPath = new NativePath(_processManager.FindGitRoot(_options.SourceRepo));
+        var targetGitRepoPath = new NativePath(_processManager.FindGitRoot(Directory.GetCurrentDirectory()));
+
+        await ValidateLocalVmr(subscription);
+        await ValidateLocalRepo(subscription);
+
+        await ValidateLocalBranchMatchesRemote(targetGitRepoPath, pr.HeadBranch);
+
+        _vmrInfo.VmrPath = string.IsNullOrEmpty(subscription.TargetDirectory)
+            ? targetGitRepoPath
+            : sourceGitRepoPath;
+
+        Console.WriteLine("Ready to execute codeflow locally. Proceed? y/n");
+
+        var confirmation = Console.ReadLine();
+
+        if (confirmation != "y")
+        {
+            Console.WriteLine("Aborting resolve operation...");
+            return;
+        }
+        else
+        {
+            Console.WriteLine("Proceeding with codeflow...");
+        }
+
+        await FlowCodeLocallyAsync(
+            targetGitRepoPath,
+            isForwardFlow: !string.IsNullOrEmpty(subscription.TargetDirectory),
+            additionalRemotes,
+            cancellationToken,
+            buildId: buildId);
+
+        Console.WriteLine("Codeflow has finished. Please resolve the conflicts and commit the changes. " +
+            "Then run git vmr resolve --continue.");
+
+        return;
+    }
+
+    private static int GetBuildIdFromTrackedPr(TrackedPullRequest pr, Guid subscriptionId)
+    {
+        if (pr.NextBuildsToApply.TryGetValue(subscriptionId, out int buildId))
+        {
+            return buildId;
+        }
+
+        throw new InvalidOperationException("Encountered an unexpected exception: could not find the build to apply. "
+            + "Please follow the instructions in the PR and seek assistance.");
+    }
+
+    #region Validations
+
+    private async Task ValidateLocalVmr(Subscription subscription)
+    {
+        var mappingName = string.IsNullOrEmpty(subscription.TargetDirectory)
+            ? subscription.SourceDirectory
+            : subscription.TargetDirectory;
+
+        var local = new Local(_options.GetRemoteTokenProvider(), _logger);
+
+        SourceManifest sourceManifest;
+        try
+        {
+            sourceManifest = await local.GetSourceManifestAsync(_vmrInfo.VmrPath);
```

Comment:
Are we sure we have the `VmrPath` set at this point? I see something setting it on line 77, which is after we're calling this method